cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -17,12 +17,669 @@
 Cask Data Application Platform Release Notes
 ============================================
 
+.. New Features
+.. Improvements
+.. Bug Fixes
+.. Known Issues
+.. API Changes
+.. Deprecated and Removed Features
+
 .. contents::
    :local:
    :class: faq
    :backlinks: none
    :depth: 2
 
+`Release 4.0.0 <http://docs.cask.co/cdap/4.0.0/index.html>`__
+=============================================================
+
+New Features
+------------
+
+#. Cask Market
+
+   - :cask-issue:`CDAP-7203` - Adds the new Cask Market view in the CDAP UI for users to
+     interact and create CDAP entities in a much easier fashion. The market hosts use cases,
+     pipelines, datapacks, drivers, Hydrator plugins, and CDAP applications as examples.
+
+#. Cask Wrangler
+
+   - :cask-issue:`WRANGLER-2` - Added Cask Wrangler: a new CDAP extension for data
```

Comment:
Can we change this to:

Adding Cask Wrangler: a new CDAP extension for interactive data preparation.